test/EzMap.IntegrationTest/PoiCollectionControllerTest.cs

```diff
@@ -4,14 +4,48 @@
 using EzMap.Domain.Dtos;
 using EzMap.Domain.Models;
 using EzMap.IntegrationTest.Infrastructure;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace EzMap.IntegrationTest;
 
 public class PoiCollectionControllerTest
 {
     [Fact]
-    public async Task CreatePoi_CorrectDataProvided_PoiShouldBeCreated()
+    public async Task CreatePoiCollection_GreaterThan100CharNameProvided_ErrorMessageShow()
+    {
+        var app = new TestWebAppFactory<Program>();
+        var client = app.CreateClient();
+        using var scope = app.Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<EzMapContext>();
+
+
+        var token = await TestHelper.GetDefaultUserToken(client);
+
+
+        var poiCol = new PoiCollectionCreateDto
+        (
+            "9VBJRFiYcF9gFeTZGSksaTMavgWTPG4Ep2pFYHqzy5i5hNDpkvsaa9VBJRFiYcF9gFeTZGSksaTMavgWTPG4Ep2pFYHqzy5i5hNDpkvsaa",
+            "citygarden"
+        );
+
+        var response = await client.RequestAsJsonAsyncWithToken(HttpMethod.Post, "api/poicollection/", token, poiCol);
+
+        Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+
+        var responseString = await response.Content.ReadAsStreamAsync();
+
+        var comparingProperty = nameof(PoiCollectionCreateDto.Name).ToLower();
+
+        var nameShowingMessage = JsonSerializer.Deserialize<ProblemDetails>(responseString, new JsonSerializerOptions()
+        {
+            PropertyNameCaseInsensitive = true
+        });
+        Assert.Contains(comparingProperty, nameShowingMessage.Errors[nameof(PoiCollectionCreateDto.Name)][0].ToLower());
```

Comment:
I thought we talked about the error assertion.

This assert only makes sure that there are error message on the property. What about the error, what if the error returned is "must be less than 50 chars", then this test still passes, but what we want - "less than 100 chars" is not satisfied.

Please do not write tests for the sake of "tests", write tests that asserts the requirement, that when the requirement is violated, the tests are what notify you firsthand.

I don't want to be pedantic, but if you can't write asserts properly on this simple test, most likely the asserts on more complicated ones are really flakey, don't write tests that are only for the sake of "having tests".